scripts/deploy_contracts.sh

```diff
@@ -18,7 +18,7 @@ deploy_contract() {
 
     echo "Originating $1 contract"
     sleep 2
-    octez_client \
+    tezos-client \
```

Comment:
I think this is fixed to be `octez-client` on latest `tezos-nix`

Reply:
Should not be in this PR I removed it